[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - LOGS_CAPTURED - Started capturing logs in process (pid: 55856).
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - chunk_blog_content - STEP_START - Started execution of step "chunk_blog_content".
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - chunk_blog_content - Loading file from: /Users/szymonzmyslony/projects/master-pipeline/pipeline/.dagster/storage/fetched_blog_content using PickledObjectFilesystemIOManager...
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - chunk_blog_content - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step chunk_blog_content
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - chunk_blog_content - LOADED_INPUT - Loaded input "fetched_blog_content" using input manager "io_manager"
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - chunk_blog_content - STEP_INPUT - Got input "fetched_blog_content" of type "Any". (Type check passed).
[32m2024-02-05 13:33:33 +0100[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - e743da8f-26e7-4faa-866b-b3467a50a66f - 55856 - chunk_blog_content - STEP_FAILURE - Execution of step "chunk_blog_content" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "chunk_blog_content"::

AttributeError: 'dict' object has no attribute 'to_dict'

Stack Trace:
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 467, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/Library/Caches/pypoetry/virtualenvs/pipeline-RtkjAAJa-py3.11/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 150, in chunk_blog_content
    serialized_last = [
                      ^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 151, in <listcomp>
    {**doc, "sections": [section.to_dict() for section in doc["sections"]]}
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/szymonzmyslony/projects/master-pipeline/pipeline/pipeline/assets.py", line 151, in <listcomp>
    {**doc, "sections": [section.to_dict() for section in doc["sections"]]}
                         ^^^^^^^^^^^^^^^
[0m
